extensions/amp-brid-player/0.1/test/amp-brid-player.out

```diff
@@ -0,0 +1,3 @@
+FAIL
```

Comment:
Should we also name these `validator-amp-brid-player.out`? Or were you planning to do this later?